src/js/stores/DCOSStore.js

```diff
@@ -146,6 +148,12 @@ class DCOSStore extends EventEmitter {
 
   }
 
+  get jobTree() {
```

Comment:
I'm not sure it makes sense to couple Jobs into this struct.
Can we abstract common functionality and have separate structs?